README.md

```diff
@@ -1019,6 +1019,262 @@ Voilà! Since we updated `index.html`, you'll need to restart the dev server. Go
 ahead and stop the `npm run serve` process, then start it again. When you open
 the browser, you should see a more stylish input field.
 
+## Step 11: Automatically fetch more data on scroll
+
+In this step, we'll implement "infinite scroll" to automatically fetch more data
+as the user scrolls to the bottom of the feed.
+
+### 11.1: Add bindings to `IntersectionObserver`
+
+We need to create bindings for `IntersectionObserver` to detect when the user
+has scrolled to the bottom of the feed.
+
+In a new file `IntersectionObserver.re`, we will be using
+[mel.send](https://melange.re/v4.0.0/communicate-with-javascript.html#calling-an-object-method)
+and
+[mel.new](https://melange.re/v4.0.0/communicate-with-javascript.html#javascript-classes):
+
+```reason
+// IntersectionObserver bindings
+type entry = {
+  isIntersecting: bool,
+  target: Dom.element,
+};
+
+type observer;
+[@mel.send] external observe: (observer, Dom.element) => unit = "observe";
+[@mel.send] external unobserve: (observer, Dom.element) => unit = "unobserve";
```

Comment:
We never use the `unobserve` binding, should we still include it?

Reply:
https://github.com/ahrefs/react-alicante-workshop/pull/9/commits/41b94a340d77019bd3ad0490327e52a6eab0826f